charts/tornjak/values.yaml

```diff
@@ -80,3 +78,9 @@ attestors:
   azure_msi:
   #   tenants:
   #   - tenant: "TENANT_ID"
+
+tornjak:
+   config:
+     enableUserMgment: true
+     authServerUri: "http://keycloak.tornjak.appdomain.cloud"
+     apiServerUri: "http://tornjak-http-tornjak.appdomain.cloud"
```

Comment:
this maybe should be frontend-config? see server-statefulset.tpl comments?

Reply:
I like this idea. I will re-shuffle the configurations. Thank you